apps/scotty/scotty.go

```diff
@@ -770,6 +891,26 @@ func httpError(w http.ResponseWriter, status int) {
 }
 
 // byEndpointHandler handles serving api/hosts requests
+type cisHandler struct {
```

Comment:
While this is being added for CIS, it's not CIS specific. This is a generic snapshot/latest feature. So, can you please use a more generic name?